test/definitions/apm/java/linux/javatomcat-debian.json

```diff
@@ -0,0 +1,56 @@
+{
+    "global_tags": {
+      "owning_team": "OpenSource",
+      "Environment": "development",
+      "Department": "Product",
+      "Product": "Virtuoso"
+    },
+    "resources": [
+      {
+        "id": "javatomcat",
+        "display_name": "AwsDebianInfraJavaInstallHost",
+        "provider": "aws",
+        "type": "ec2",
+        "size": "t2.micro",
+        "ami_name": "debian-10-amd64-2020????-???-*",
+        "user_name": "admin"
+      }
+    ],
+    "services": [
+      {
+        "id": "java1",
+        "source_repository": "https://github.com/newrelic/demo-javatron.git",
+        "deploy_script_path": "deploy/debian/roles",
+        "port": 6001,
+        "destinations": ["javatomcat"],
+        "files": [
+          {
+            "destination_filepath": "javatron/cronjob.json",
+            "content": [
+              {
+                "frequency": "* * * * *",
+                "job": "/bin/bash -c \"for i in {1..30}; do curl '[service:java1:url]/'; curl '[service:java1:url]/api/inventory'; sleep 1; done\""
+              }
+            ]
+          }
+        ]
+  
+      }
+    ],
+    "instrumentations": {
+      "resources": [
+        {
+          "id": "javatomcat",
+          "resource_ids": ["javatomcat"],
+          "provider": "newrelic",
+          "source_repository": "-b java_linux2_apm_agent https://github.com/newrelic/open-install-library.git",
```

Comment:
Before this PR gets merged we need to remove all the reference to this branch in the recipes.